test/EasyRdf/Sparql/ClientTest.php

```diff
@@ -462,6 +462,82 @@ public function testAcceptHeaders()
         $this->assertContains('text/turtle', $types);
     }
 
+    /**
+     * Test for issue https://github.com/sweetyrdf/easyrdf/issues/9
+     *
+     * Sparql\Client calls $client->setHeaders with a name-value pair, regardless of
+     * whether $client is of type EasyRdf\Http\Client or Zend\Http\Client.
+     */
+    public function testIssue9EasyRdfHttpClient()
+    {
+        /*
+         * setup mocks
+         */
+        // response
+        $response = $this->getMockBuilder(\EasyRdf\Http\Response::class)->disableOriginalConstructor()->getMock();
+        $response->method('getStatus')->willReturn(204); // no content
+        $response->method('isSuccessful')->willReturn(true);
+
+        // client
+        $easyRdfHttpClient = $this->getMockBuilder(\EasyRdf\Http\Client::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $easyRdfHttpClient->method('request')->willReturn($response);
+        $easyRdfHttpClient
+            ->expects($this->exactly(1))
+            ->method('setHeaders')
+            // we only check first parameter, which has to be 'Accept'
+            ->with($this->callback(function ($firstParam) {
+                return 'Accept' == $firstParam;
+            }));
+
+        Http::setDefaultHttpClient($easyRdfHttpClient);
+
+        $this->sparql = new Client('http://localhost:8080/sparql');
+
+        $this->sparql->query('test');
+    }
+
+    /**
+     * Test for issue https://github.com/sweetyrdf/easyrdf/issues/9
```

Comment:
Please can you update this to:
https://github.com/easyrdf/easyrdf/issues/305

Reply:
Done.